### DISCUSSION TITLE
account suspended for no reason

### DISCUSSION THREAD

**Discussion by gpereiradev** [community/community#116564] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

My github business account was suspended for no reason. It is directly impacting my work. I need the account to be reestablished urgently
![conta suspensa](https://github.com/community/community/assets/29125964/a7b90baf-50b1-41d1-8358-3680ef64e454)

Ticket Support ID 2708358

**Comment by arvindrag** [comment]
I'm having the same issue: https://github.com/orgs/community/discussions/116561

**Comment by pooksh** [comment]
I had a similar issue, suspended for no reason, but was able to change my password and login again after submitting a ticket about not being able to login due to suspension. https://github.com/orgs/community/discussions/116566

**Comment by gpereiradev** [comment, marked as answer]
try logging in again, it worked

**Comment by github-staff** [comment]
Thank you for your patience. Our Platform Health team has investigated this issue and found the root cause to be an error in how malicious behavior was flagged, causing the suspension of several accounts. This error has been resolved and the affected accounts have been reinstated.
We apologize for the inconvenience this caused. If your account is still currently suspended, please reach out to Support for further investigation.

**Reply by HimanshuW4** [reply]
I have the same issue: https://github.com/orgs/community/discussions/116910. But nothing has changed so far. How much more time is it's gonna take to get the rest of the accounts reinstated? @github-staff

**Comment by alanzimm85** [comment]
Please help, i need to get my account back, i sent a lot of curriculums with projects on this github account. I didn´t do nothing wrong. It's not from this account, i can´t acces the other. Ticket ID: 2921645

You have been suspended from GitHub.
 
This moderation action was taken after we received a report and determined that your account violates our Acceptable Use Policy prohibiting the use of our platform in direct support of unlawful attacks that cause technical harms. We do not allow anyone to use our platform in direct support of unlawful attacks that cause technical harms, such as using GitHub as a means to deliver malicious executables or as attack infrastructure, for example by organizing denial of service attacks or managing command and control servers. Technical harms means overconsumption of resources, physical damage, downtime, denial of service, or data loss, with no implicit or explicit dual-use purpose prior to the abuse occurring.

When making content moderation decisions, we consider information from a variety of sources, including: account profile data, information contained in submitted reports/notices or discovered through our own voluntarily initiated investigations, and context around the contents of the repository.

If you wish to regain access or would like to dispute that a violation occurred and can provide additional information to show that a different decision should have been reached, please review our [Appeal and Reinstatement Policy](https://docs.github.com/en/site-policy/acceptable-use-policies/github-appeal-and-reinstatement) and submit a request via our [form](https://support.github.com/contact/reinstatement).
